byway/src/main/webapp/recommendations.js

```diff
@@ -134,18 +97,19 @@ function findRegions(directionResult) {
 /**
  * Set a timeout to delay the browser.
  * @param {Number} delayMs number of milliseconds
- * @return empty promise after delayMs milliseconds passed
+ * @return fulfilled promise after delayMs milliseconds passed
  */
 function delayPromise(delayMs) {
   return new Promise(resolve => setTimeout(resolve, delayMs));
 }
 
 /**
- * Go through a user's interests and search for places
- * fitting those interests. Search around the regions
- * previously found using textSearch. Prioritize finding
- * past results loaded through sessionStorage to avoid calling
- * textSearch repeatedly from the PlacesService.
+ * Go through a trip's interests and regions to find relevant places
+ * using textSearch from PlacesService. Prioritize finding past results
+ * loaded through sessionStorage to avoid making multiple calls to
+ * textSearch. Additionally, set an intermediate timeout between calls
+ * to findPlacesWithTextSearch with delayPromise to avoid hitting a query limit.
+ * Alert the user of any non-OK status codes from PlacesService.
```

Comment:
nit: make comment more succinct, you already kind of explain this stuff inside the function.

Reply:
Oh right, I updated the function description with the comments inside the function, but forgot to remove. Updated